frontend/docs/skills/track-backend-issues.md

```
@@ -0,0 +1,257 @@
+
+- **Existing frontend tracking issues:** issues that have the
+  `area/frontend` label OR whose title starts with `[frontend]`. Either
+  signal identifies a tracking issue (the title prefix is the fallback
+  when labels can't be applied due to permissions).
+- **Backend issues:** all remaining issues. These are candidates for
+  analysis.
+
+Also fetch closed frontend tracking issues — these may need updating if
+their corresponding backend issue was recently closed:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)